release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -77,6 +77,16 @@ Platform (GCP) using user-provisioned infrastructure and a shared VPC.
 
 // For more information, see ../installing//installing_gcp/installing-gcp-user-infra-vpc.adoc#installing-gcp-user-infra-vpc[Installing a cluster on GCP using Deployment Manager templates and a shared VPC].
 
+[id="ocp-4-5-three-node-bare-metal-deployments"]
+==== Three-node bare metal deployments
```

Comment:
Just a question on if "deployment" is the correct term here, or if it should be installation or cluster?

Reply:
I copied this title from the section for 4.2 when it was announced as TP [here](https://docs.openshift.com/container-platform/4.2/release_notes/ocp-4-2-release-notes.html#ocp-4-2-three-node-bare-metal-deployments). It's also listed as such in the TP table. From what I can tell, that is the correct term to refer to the topic.